csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -241,9 +241,19 @@ module Public {
     }
 
     /**
-     * Holds if the summary is auto generated.
+     * Holds if the summary is auto generated and not manually generated.
      */
     predicate isAutoGenerated() { none() }
+
+    /**
+     * Holds if the summary is manually generated and not auto generated.
+     */
+    predicate isManuallyGenerated() { none() }
```

Comment:
I think I prefer a leaner interface, since these two predicates are only for use in the metric query. How about this?
```
predicate hasProvenance(boolean generated) { summaryElement(this, _, _, _, generated) }
```

Reply:
Sure, I can switch to that. Updated in commit https://github.com/github/codeql/pull/11585/commits/623068c4b9b0ae1edbab97b9ed4a317f7671625d.